strapi-react-app/react-frontend/src/components/lint-test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
what the purpose of this file?

Reply:
Removed this file, just updated package.json and package-lock.json to have it working locally